packages/interactivity/src/directives.tsx

```diff
@@ -600,7 +600,12 @@ export default () => {
 		}: {
 			element: any;
 		} ) => {
-			// Preserve the initial inner HTML.
+			// Shown deprecation warning
+			warn(
+				'The "data-wp-ignore" directive is deprecated and will be removed in a future WordPress release.'
```

Comment:
Let's make this message consistent with the output of the `deprecated()` function from `@wordpress/deprecated`. 🙂 

There are other packages using `deprecated()`, like [here](https://github.com/WordPress/gutenberg/blob/ff4a970456a7b00d0c6256950a544d6e9d6138d8/packages/blocks/src/store/actions.js#L68-L70), but we cannot use it inside the Interactivity API module.

```suggestion
				'The "data-wp-ignore" directive of the Interactivity API is deprecated since version 6.9 and will be removed in version 7.0.'
```

Reply:
Addressed in https://github.com/WordPress/gutenberg/pull/70945/commits/78aa484925a1da5a2b385236087bb0ce139f9e1f. Thanks!